lib/support/util.js

```diff
@@ -14,5 +16,45 @@ module.exports = {
           formatted += ` (±${fmt(metric.mdev)})`;
       }
       return formatted;
+  },
+  getResultLogLine(result, options) {
+    // don't bother if no statistics or silent x2
+    if (!result.statistics || !result.statistics.timings || !result.statistics.timings.pageTimings || options.silent > 1) return result;
+
+    let totalSize = 0;
+    let requests = result.browserScripts[0].timings.resourceTimings.length + 1;
+
+    if (result.har && result.har.log.pages[0] !== undefined) {
```

Comment:
Could we drop the log.pages[0] check? If we're including malformed hars in the result we should fix that bug elsewhere, rather than checking for it here.